pages/national_resources/no_resources.md

```diff
@@ -242,6 +242,21 @@ Sikt is a Norwegian governmental agency that provides [(sensitive) personal data
 
 NRIS, a collaboration between Sigma2 (a subsidiary of Sikt) and four universities, gives [technical and administrative support](https://www.sigma2.no/user-support) to researchers in Norway on large-scale data storage and high-performance computing.
 
+
+### Communities for research data management support personell
+
+* [Life-Science RDM Group](https://elixir.no/organization/biomeddata/Life-Science-RDM-Group)
+* [RDA Norway](https://www.rd-alliance.org/groups/rda-norway)
+* [University of Oslo Data Manager Network](https://www.ub.uio.no/english/libraries/dsc/data-managers-network/)
+* [Bergen Research Data Network](mailto:research-data@uib.no?subject=forskningsdata@uib.no)
```

Comment:
For Bergen Research Data Network, do you mean to link this https://www.uib.no/en/research/143544/research-infrastructure-network ? here, two email addresses are mentioned..

Reply:
no, this is 2 different networks